app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,18 +1,21 @@
-
 - if @logs.empty?
   = content_tag(:div, t('common.no_records'), class: 'alert alert-info')
 - else
   .show-actions.hidden-print
     a data-expand-all=".logs-index" href="#" = t('common.expand_all')
-    a data-hide-all=".logs-index" href="#" = t('common.hide_all')
+    - if @step && @context && @context == "calendar"
+      a data-hide-all=".logs-index" href="#" = t('common.condense_all')
+    - else
+      a data-hide-all=".logs-index" href="#" = t('common.hide_all')
 
-  - if @step
-    h5
-      - if @step.milestone?
-        i.fa.fa-fw.fa-flag-o
-      - elsif @step.checkin?
-        i.fa.fa-fw.fa-calendar
-      = @step.summary
+  - unless @context && @context == "calendar"
+    - if @step
+      h5
+        - if @step.milestone?
+          i.fa.fa-fw.fa-flag-o
+        - elsif @step.checkin?
+          i.fa.fa-fw.fa-calendar
```

Comment:
I will look and see. (I didn't add this code in this feature. It already existed. I just moved the code inside an if statement.)